pkg/core/resources/apis/mesh/dataplane_helpers.go

```diff
@@ -84,3 +105,14 @@ func (d *DataplaneResource) GetIP() string {
 	}
 	return ifaces[0].DataplaneIP
 }
+
+func (d *DataplaneResource) GetProtocol(idx int) Protocol {
```

Comment:
How about `idx` -> `inboundIdx`? I had to read the func body to know what is it about.

Reply:
done